logging/Limiting_Access_to_Logs.md

```diff
@@ -252,4 +267,19 @@ process:
 |V4MCLUSTER_ADMIN_kibana_users |    | Provides an easy way to link the above roles with a user.
 
 Linking users to the back-end role V4MCLUSTER_ADMIN_kibana_users via the 
-Kibana security plug-in grants those users the access described in the table above.
\ No newline at end of file
+Kibana security plug-in grants those users the access described in the table above.
+
+You can also use the user.sh script to grant a user the same level of access as 
+a `logadm` user. See the following syntax for an example:
+
+<pre>
+logging/bin/user.sh CREATE --user <i>user_name</i> --password <i>password</i> 
+--namespace <i>namespace</i> --tenant <i>tenant</i>
+</pre>
+
+Where: 
+
+- *user_name* is "chen".
+- *password* is the password to sign in to Kibana as the user "chen".
+- *namespace* is `_all_`.
+- *tenant* is `_all_`.
```

Comment:
If this example is intended to show how to create a user with `logadm`-like access, let's revise it to show that specific example rather than generic syntax.  So, replace the placeholder values with the actual values shown in the WHERE section.  (Use 'chenspassword' or something for the password.)

Reply:
Same question as above. Since you're more definitive in this comment, I'll make it a pure example.

Reply:
Revised content:

You can also use the user.sh script to grant a user the same level of access as 
a `logadm` user. For example, to grant the user "chen" this level of  
access controls, submit the following command:

<pre>
logging/bin/user.sh CREATE --user chen --password chenspassword 
--namespace _all_ --tenant _all_
</pre>